README.md

```diff
@@ -32,27 +36,38 @@ PS> git clone https://github.com/sublimehq/Packages.git
 PS> cmd /c mklink /J "$env:APPDATA/Sublime Text 3/Packages/Python" (convert-path ./Packages/Python)
 ```
 
-Alternatively, download the portable version, and clone this repository directly as a subdirectory of the *Data* folder.
+Alternatively, download the portable version, and clone this repository directly as a subdirectory of the `Data` folder.
+
+### ⚠️ Things to keep in mind
+
+After you've finished, keep in mind that you're now overriding a default package.
+When Sublime Text updates to a new version, you'll run the risk of having an out-of-date package unless you pull the latest changes from this repository.
+
+## Reference
 
----
+Please refer to the **official docs**:
 
-After you've finished, keep in mind that you're now overriding a default package. When Sublime Text updates to a new version, you'll run the risk of having an out-of-date package unless you pull the latest changes from this repository.
+* [syntax definitions](https://www.sublimetext.com/docs/syntax.html#ver-dev)
+* [scope naming](https://www.sublimetext.com/docs/scope_naming.html)
 
 ## Pull Requests
 
 Pull requests should:
 
  1. Start with the name of the package in square brackets, e.g. `[XML]`.
- 2. Be as small and self-contained as is reasonable. *Smaller changes are quicker to review.*
- 3. Include a new, or enhanced, [syntax test](http://www.sublimetext.com/docs/syntax.html) when changing a `.sublime-syntax` file.
+ 2. Be as small and self-contained as is reasonable.
+    *Smaller changes are quicker to review.*
+ 3. Include a new, or enhanced, [syntax test](https://www.sublimetext.com/docs/syntax.html#testing) when changing a `.sublime-syntax` file.
  4. Have multiple syntax tests and a set of performance measurements (old vs. new) when making significant changes.
 
 ### New Packages
 
-Pull requests for new packages won't be accepted at this stage, as new packages can cause issues for users who have a package with the same name installed via Package Control. There are some planned changes that will address this in the future.
+Pull requests for new [packages](https://www.sublimetext.com/docs/packages.html) won't be accepted at this stage, as new packages can cause issues for users who have a package with the same name installed via [Package Control](https://packagecontrol.io/).
+There are some planned changes that will address this in the future.
```

Comment:
Is this last sentence still valid?

Reply:
I guess we can assume there won't be any feedback from SublimeHQ in the near future about this.